setup.py

```diff
@@ -40,6 +40,18 @@ def _find_msbuild_tool(tool="msbuild.exe", use_windows_sdk=False):
                 locappdir, vcpy27, r"x64\mt.exe")
             if os.path.exists(mtpath):
                 return mtpath      
+        if sys.version_info[:2] == (3,5):
```

Comment:
There shouldn't be any need for hardcoded paths. Not everyone will have installed the windows SDK on the C drive.